src/app/src/main/kotlin/org/vaccineimpact/api/app/ErrorHandler.kt

```diff
@@ -32,15 +30,12 @@ open class ErrorHandler(private val logger: Logger = LoggerFactory.getLogger(Err
         val error = when (exception)
         {
             is MontaguError -> exception
-            else ->
-            {
-                logger.error(unhandledExceptionMessage, exception)
-                UnexpectedError()
-            }
+            is ValidationException -> ValidationError(exception.errors)
```

Comment:
can we then remove the catch and re-throw of `ValidationException` as `ValidationError` that happens elsewhere?